pkgs/top-level/all-packages.nix

```diff
@@ -10826,6 +10826,7 @@ with pkgs;
   simp_le = callPackage ../tools/admin/simp_le { };
 
   sfml = callPackage ../development/libraries/sfml { };
+  csfml = callPackage ../development/libraries/csfml { };
```

Comment:
The alphabetical order is usually preferred.

Reply:
I figure it's a good idea to group these together because `csfml` is just a binding for `sfml`, much like `cppzmq` and `czmq` for `zeromq`. I can definitely make this change if it's a strong preference though.